src/main/java/gift/controller/WishListController.java

```diff
@@ -0,0 +1,82 @@
+package gift.controller;
+
+import gift.model.GiftResponse;
+import gift.model.User;
+import gift.model.UserGift;
+import gift.model.WishResponse;
+import gift.service.GiftService;
+import gift.service.WishService;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+public class WishListController {
+
+    private final WishService wishService;
+    private final GiftService giftService;
+
+    @Autowired
+    public WishListController(WishService wishService, GiftService giftService) {
+        this.wishService = wishService;
+        this.giftService = giftService;
+    }
+
+    @GetMapping("/wish")
+    public ResponseEntity<?> getGiftList(HttpServletRequest request) {
```

Comment:
다음과 같이도 값을 가져올 수 있습니다ㅎㅎ
```suggestion
    public ResponseEntity<?> getGiftList(@RequestAttribute("user") User user) {
```

Reply:
이런 방법도 있었던건 몰랐네요.이게 더 가독성 측면에서 좋은 방법인 것 같습니다!